src/d3d9/d3d9_device.h

```diff
@@ -983,13 +983,6 @@ namespace dxvk {
       return m_isD3D8Compatible;
     }
 
-    void SetD3D8CompatibilityMode(bool compatMode) {
```

Comment:
I like the change to just set that once at the start.

Reply:
Yeah, not to mention the old way of doing things allowed toggling compatibility state during a device's lifecycle, which makes no sense.